scripts/buildTokens.js

```diff
@@ -0,0 +1,118 @@
+const glob = require('glob')
+const StyleDictionary = require('style-dictionary')
+const {
+  transformScssTokens,
+  transformTailwindTokens
+} = require('./utils/transformTokens')
+
+const prefix = 'dp-'
+
+const tokensPath = 'tokens/src/**/*.json'
+
+const files = glob
+  .sync(tokensPath)
+  .map(filePath => filePath
+    .replace('tokens/src/', '')
+    .replace('color/', '')
+    .replace('.json', ''))
+    .filter(filePath => !filePath.startsWith('_')) // Do not render tokens only used internally
+
+// Define custom output groups for Tailwind, rather than just grouping them by file path
+const corePlugins = ['fontSize', 'boxShadow', 'breakpoints', 'rounded', 'space', 'zIndex']
+const corePluginsColor = ['color', 'textColor', 'backgroundColor', 'borderColor']
```

Comment:
By now I can't see the point why its necessary to divide corePlugins and corePluginsColor.